### ISSUE TITLE
Opa policy bundle with optimizations and future.keywords

### ISSUE THREAD

**Issue by keshavprasadms** [open-policy-agent/opa#4111]
## Short description
Opa version: 0.34.2
When I use `import future.keywords.in` in a opa policy and then bundle it with optimization as 1 or 2, opa doesn't start and complains with an error. Please see below to reproduce.

`main.rego`
```
package main

allow {
  data.policy.valid
}
```

`policy.rego`
```
package policy

import future.keywords.in

valid {
  input.message in ["abc", "def"]
}
```
Creating the bundle with optimization level 0 works
```
opa build --bundle . --optimize 0 --entrypoint "main/allow"
opa run -s -b bundle.tar.gz
curl "localhost:8181/v1/data/main/allow" --data '{"input": {"message": "abc"}}'
{"result":true}
```

But creating the bundle with optimization level 1 or 2 doesn't start opa
```
opa build --bundle . --optimize 1 --entrypoint "main/allow"
opa run -s -b bundle.tar.gz 
error: load error: bundle bundle.tar.gz: 1 error occurred: /optimized/policy.rego:5: rego_parse_error: unexpected ident token: expected \n or ; or }
	__local0__2 in ["abc", "def"]
	            ^
```

Could you please tell me if I am doing something wrong? How can I achive optimizations 1 or 2 using future.keywords as imports.

**Comment by anderseknert** [comment]
Hi @keshavprasadms! And thanks for filing this. I don't think you're doing anything wrong. That does indeed look like a bug.

**Comment by keshavprasadms** [comment]
@anderseknert Any info on when this can get fixed? Will the fix be available only in future release or will there be a patch to 0.34.x?

**Comment by tsandall** [comment]
@keshavprasadms @srenatus is working on this right now. The fix will be released in v0.36.0 at the end of the month. If you need this backported, we could _potentially_ cherry-pick it into v0.35.
